src/Dockerfile.template

```diff
@@ -31,6 +35,26 @@ ENV BASH_ENV=${APP_ROOT}/etc/scl_enable \
 {% endif %}
 
 {{ macros.env_metadata() -}}
+{% if spec.el_version == '7' %}
+ENV SUMMARY="Platform for building and running Python $PYTHON_VERSION applications" \
```

Comment:
Šlo by summary a description udělat tak, aby nebyly v šabloně duplicitně?

Reply:
ok zkusim to, ty testy už vím o co jde opravim to zaroven